app/javascript/packages/components/full-screen.tsx

```diff
@@ -111,7 +111,7 @@ function FullScreen(
       {!hideCloseButton && (
         <button
           type="button"
-          aria-label={t('users.personal_key.close')}
+          aria-label={t('account.navigation.close')}
```

Comment:
This should probably have its own translation key, so we don't run into similar future issues like here where we're referencing personal key long after the personal key modal has been removed.